dev-example/App.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div>
     <form-wizard @on-complete="onComplete"
+                 :hide-buttons="false"
```

Comment:
It's more natural to say that true for hide-buttons actually hides buttons than the other way around